src/explore-education-statistics-admin/src/components/editable/EditableEmbedForm.tsx

```diff
@@ -20,6 +21,15 @@ interface Props {
   onSubmit: (values: EditableEmbedFormValues) => void;
 }
 
+const errorMappers = [
```

Comment:
Nitpick, but we call this `errorMappings` in all other places of the admin app.

Reply:
No probs, good spot!

Reply:
Renamed - I must've gotten my steer from a similar name in `useFormSubmit()`.